### DISCUSSION TITLE
Is it possible to have a cut distribution/limit min values?

### DISCUSSION THREAD

**Discussion by wverestek** [neperfepx/neper#319] [Q&A] [answered]
Hi all, 

i was wondering if it is possible to have a cut distribution. The reason is that my model could become quite big and i would like to e.g. limit the minimal grain size to avoid very tiny elements. I was thinking if it is possible to use sth like:

neper -T -n 1000 -morpho "diameq:normal(1,0.2):valmin(0.5)"

which obviously does not work because I am writing here. So, if there are any ideas floating around, please let me know.

Regards
Wolfgang

**Comment by rquey** [comment, marked as answer]
You were close:
```
$ neper -T -n 1000 -morpho "diameq:normal(1,0.2,from=0.5)"
```

**Reply by wverestek** [reply]
Somehow I managed to not find that in the documentation. Thanks a lot :)

**Reply by smallrain1996** [reply]
In
The documentation for Neper 4.2.0
A.1.4 Statistical Distributions
